docs/editors/debug-adapter-protocol.md

```diff
@@ -57,6 +58,10 @@ Apart from using code lenses, users can start a debug session by executing the
 identically named classes in different modules. A uri will be returned that can
 be used by the DAP client.
 
+`envFile` is an optional parameter, which allows you to specify a path to a
```

Comment:
Let's add information that provided path can be relative to root workspace (or absolute path - absolute path will work too?)

Reply:
Both relative and absolute paths are supported. I've mentioned about this in the docs as well as created an additional test to cover this (10dc7e7).